showItemPageBE/src/main/java/com/talentpath/shamazin/showItemPage/Models/Question.java

```diff
@@ -0,0 +1,84 @@
+package com.talentpath.shamazin.showItemPage.Models;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name="questions")
+public class Question {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer questionId; // Creates a unique id for each question asked in the DB
+
+    private Integer itemId;     // Associate the question to a particular item/product
+
+    private Integer userId;     // Associate the question with a user / confirmed buyer
+
+    private String question;    // The question that will be displayed to the user
+
+    private Integer votes;      // To store the number of up/downvotes a question receives
+
+    private LocalDate date;     // Store the date of when the question was asked
+
+    public Question(){
+
+    }
+
+    public Question(Question that){
```

Comment:
this is just my general nitpick, so it's totally fine to just leave it as is tbh, but i think renaming "that" to something like "question" would make the object we're referring to a lot more clear and overall readable. although, i suppose "question.question" looks a bit silly, so it's up to you. i think we should just make sure everyone else is on the same page if this is the naming convention we decide to go with

Reply:
I was just looking up some old code form training & found that we used 'that' in constructors in some old cold for hangman, etc. so figured I'd go with that but i can always change it

Reply:
ahh, got it. works for me then!